pkgs/stdenv/freebsd/bootstrap-files/bootstrapFiles.nix

```diff
@@ -0,0 +1,99 @@
+{ system }: ((import <nixpkgs> {}).callPackage ({ stdenv, pkgsCross, runCommand, lib, stdenvAdapters, buildPackages, ... }:
```

Comment:
Supposedly we need to make sure that hydra provides these bootstrap files soon?
Needs to be coordinated with the hydra team.